src/routes/RouteWrapper.tsx

```diff
@@ -1,17 +1,21 @@
-import React, { memo } from "react";
+import React, { FC, memo } from "react";
 import { Route, Redirect } from "react-router-dom";
 import DefaultLayout from "layouts/DefaultLayout/DefaultLayout";
+import { Permission } from "features/permissions/permissions";
+import { RouteItemDef } from "types/routeDef";
+import { AUTH_ROUTE, ROOT_ROUTE } from "./Routes.config";
 
-const RouteWrapper: React.ElementType = ({
-  component,
+const RouteWrapper: FC<RouteItemDef> = ({
+  component: Component,
```

Comment:
same as above, capitalize `C`

Reply:
But a prop cannot be capitalized, right @agrosner? 🤔  which is why we would do this to tell React that it is actually a component

Reply:
props can be capitalized, but discard my comments. should be fine.